src/style.css

```diff
@@ -0,0 +1,139 @@
+/* #region page */
+.page {
```

Comment:
you can use just `html` for setting styles
```suggestion
html {
```